style.css

```diff
@@ -1,450 +1,1097 @@
+    padding-top: 10%;
+  }
+
+  .text {
+    margin-block-start: 0;
+    text-align: initial;
+    width: 100%;
+    font-family: "Poppins", sans-serif;
+    font-size: 48px;
+    line-height: 60px;
+    letter-spacing: 0.37px;
+    color: #091e42;
+  }
+
+  .text2 {
+    width: 95%;
+    text-align: initial;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    padding-right: 20%;
+    line-height: 28px;
+    height: auto;
+    color: #344563;
+  }
+
+  .a {
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 24px;
+    color: #7f8cff;
+    margin-top: 3%;
+    display: flex;
+    padding-left: 15px;
+    align-self: baseline;
+    text-decoration: none;
+    margin-bottom: 2%;
+  }
+
+  .social-media {
+    display: flex;
+    align-items: center;
+    max-width: 40%;
+  }
+
+  .c001 > img,
+  .card2001 > img,
+  .card3001 > img,
+  .card4001 > img {
+    width: 100%;
+    height: auto;
+  }
+
+  .social-media > li > a > img {
+    height: 20px;
+    font-size: 2px;
+  }
+
+  .cards1,
+  .card2,
+  .swapcard3,
+  .card4 {
+    display: flex;
+    align-items: center;
+    flex-direction: row;
+    width: 90%;
+    border-radius: 8px;
+    gap: 5%;
+    background-color: white;
+    margin: 3% auto;
+    padding: 12px 3%;
+  }
+
+  .c001,
+  .card2001,
+  .card3001,
+  .card4001 {
+    width: 65%;
+  }
+
+  .card2001,
+  .card4001 {
+    order: 2;
+  }
+
+  .coo2 > {
+    width: 40%;
+  }
+
+  ul {
+    display: flex;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    list-style: none;
+    color: #344563;
+  }
+
+  h3 {
+    color: #344563;
+    margin-right: 7px;
+  }
+
+  p {
+    width: 295px;
+    height: 76px;
+    margin: 8px 7px;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 20px;
+    line-height: 24px;
+    color: #344563;
+  }
+
+  .lang {
+    float: left;
+    width: 100px;
+    height: 16px;
+    list-style: none;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 16px;
+    line-height: 16px;
+    letter-spacing: 0.03em;
+    color: #6070ff;
+  }
+
+  h5 {
+    background-color: #ebebff;
+    padding: 4px 12px;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    border-radius: 8px;
+  }
+
+  .button {
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 17px;
+    line-height: 24px;
+    display: flex;
+    align-items: center;
+    text-align: center;
+    letter-spacing: 0.03em;
+    color: #396df2;
+    box-sizing: border-box;
+    width: 150px;
+    padding: 12px 3px;
+    background: #fff;
+    border: 1px solid #435c9d;
+    border-radius: 14px;
+    margin-top: 3rem;
+    transition: color 0.5s ease;
+  }
```

Comment:
- [ ] I can see that you put some hover on the button but isn't applied kindly fix this for your project to look attractive and recruiters can reach you out for the good work you did :wink: 
your design
![butttttt](https://user-images.githubusercontent.com/88894525/206873685-a25e08bd-d25c-410c-bfe4-f1e2ba77452f.png)

Figma design 
![goot](https://user-images.githubusercontent.com/88894525/206873751-c08a86d9-efc3-4e1f-9b94-55d9651f3a68.png)